api/src/pcapi/routes/backoffice/collective_offers/collective_offers_blueprint.py

```diff
@@ -642,22 +684,27 @@ def reject_collective_offer(collective_offer_id: int) -> utils.BackofficeRespons
     form = forms.RejectCollectiveOfferForm()
     if not form.validate():
         flash(utils.build_form_error_msg(form), "warning")
+        if utils.is_request_from_htmx():
+            return _render_collective_offers()
```

Comment:
Pourquoi recharger toutes les offres collectives lorsqu'il y a un échec de rejet d'une seule offre ?
```suggestion
            return _render_collective_offers([collective_offer_id])
```

Reply:
c'est l'inverse, on en recharge aucune (puisqu'on a rien touché il est inutile de recharger des offres)